core/src/main/scala/org/dbpedia/extraction/config/dataparser/FlagTemplateParserConfig.scala

```diff
@@ -748,6 +768,513 @@ object FlagTemplateParserConfig
+              "CAF"->"República Centroafricana",
+              "CPV"->"Cabo Verde",
+              "REU"->"Reunión",
+              "RUS"->"Rusia",
+              "ROU"->"Rumanía",
+              "RWA"->"Ruanda",
+              "ESH"->"República Árabe Saharaui Democrática",
+              "KNA"->"San Cristóbal y Nieves",
+              "LCA"->"Santa Lucía",
+              "VCT"->"San Vicente y las Granadinas",
+              "BLM"->"San Bartolomé",
+              "MAF"->"San Martín",
+              "SPM"->"San Pedro y Miquelón",
+              "SLV"->"El Salvador",
+              "ASM"->"Samoa Estadounidense",
+              "WSM"->"Samoa",
+              "SMR"->"San Marino",
+              "SEN"->"Senegal",
+              "SRB"->"Serbia",
+              "SYC"->"Seychelles",
+              "SLE"->"Sierra Leona",
+              "SGP"->"Singapur",
+              "SXM"->"Isla de San Martín",
+              "SVK"->"Eslovaquia",
+              "SVN"->"Eslovenia",
+              "SOM"->"Somalia",
+              "LKA"->"Sri Lanka",
+              "USA"->"Estados Unidos",
+              "SWZ"->"Suazilandia",
+              "SDN"->"Sudán",
+              "SUR"->"Surinam",
+              "SJM"->"Svalbard y Jan Mayen",
+              "SYR"->"Siria",
+              "CHE"->"Suiza",
+              "SWE"->"Suecia",
+              "TJK"->"Tayikistán",
+              "THA"->"Tailandia",
+              "TZA"->"Tanzania",
+              "TLS"->"Timor Oriental",
+              "TGO"->"Togo",
+              "TKL"->"Tokelau",
+              "TON"->"Tonga",
+              "TTO"->"Trinidad y Tobago",
+              "TUN"->"Túnez",
+              "TUR"->"Turquía",
+              "TKM"->"Turkmenistán",
+              "TCA"->"Islas Turcas y Caicos",
+              "TUV"->"Tuvalu",
+              "UGA"->"Uganda",
+              "UKR"->"Ucrania",
+              "URY"->"Uruguay",
+              "UZB"->"Uzbekistán",
+              "VUT"->"Vanuatu",
+              "WLF"->"Wallis y Futuna",
+              "VAT"->"Ciudad del Vaticano",
+              "VEN"->"Venezuela",
+              "HUN"->"Hungría",
+              "GBR"->"Reino Unido",
+              "VNM"->"Vietnam",
+              "ITA"->"Italia",
+              "CIV"->"Costa de Marfil",
+              "BVT"->"Isla Bouvet",
+              "CXR"->"Isla de Navidad",
+              "IMN"->"Isla de Man",
+              "SHN"->"Santa Elena, Ascensión y Tristán de Acuña",
+              "ALA"->"Islas de Aland",
+              "COK"->"Islas Cook",
+              "VIR"->"Islas Vírgenes de los Estados Unidos",
+              "HMD"->"Islas Heard y McDonald",
+              "CCK"->"Islas Cocos",
+              "MHL"->"Islas Marshall",
+              "FRO"->"Islas Feroe",
+              "SLB"->"Islas Salomón",
+              "STP"->"Santo Tomé y Príncipe",
+              "ZMB"->"Zambia",
+              "ZWE"->"Zimbabue",
+              "ARE"->"Emiratos Árabes Unidos"
+            ), //end es
+        "fr" ->
```

Comment:
this time the diff was too big for me to notice that :) I saw that you put es in order but then noticed fr